### PR TITLE
Added pretty printing for indexing using unicode

### PR DESCRIPTION
**Pull request by SpringVaS**
Firstly, now variables containing indices (like `let $decr_1 :: ` ...) also pretty printed with unicode indices. (`$decr₁`).

New: multiple indices (like `$decr_1_1`) are displayed as (`$decr₁_₁`). The dafny parser is adapted accordingly and can handle these pretty printed expressions in the proof script.

### CODE REVIEW

File: modules/core/res/META-INF/services/edu.kit.iti.algover.term.prettyprint.PrettyPrintExtension [hunk 1]
```diff
@@ -7,4 +7,4 @@ edu.kit.iti.algover.term.prettyprint.HeapStorePrinterExtension
 edu.kit.iti.algover.term.prettyprint.SequencePrinterExtension
 edu.kit.iti.algover.term.prettyprint.ExtensionsPrinterExtension
 edu.kit.iti.algover.term.prettyprint.DafnyFunctionPrinterExtension
-edu.kit.iti.algover.term.prettyprint.SubscriptPrinterExtension
```
Comment (mattulbrich) [inline modules/core/res/META-INF/services/edu.kit.iti.algover.term.prettyprint.PrettyPrintExtension:10]: This class can probably be removed then.

File: modules/core/src/edu/kit/iti/algover/term/prettyprint/MultiSubscriptPrinterExtension.java [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+import edu.kit.iti.algover.term.ApplTerm;
+import edu.kit.iti.algover.term.FunctionSymbol;
+import edu.kit.iti.algover.term.VariableTerm;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class MultiSubscriptPrinterExtension implements PrettyPrintExtension, VariablePrettyPrintExtension {
+
+    private static final String SUBSCRIPT_PATTERN = "(.*)_([0-9]+)";
```
Comment (mattulbrich) [inline modules/core/src/edu/kit/iti/algover/term/prettyprint/MultiSubscriptPrinterExtension.java:12]: This pattern does not look right. Are there test cases for this pretty printing?

**Review by mattulbrich — CHANGES_REQUESTED** [review]
Test cases for the pretty printer would be nice. I think it may fail.

File: modules/core/src/edu/kit/iti/algover/term/prettyprint/MultiSubscriptPrinterExtension.java (hunk 2 above)
Comment (SpringVaS) [reply modules/core/src/edu/kit/iti/algover/term/prettyprint/MultiSubscriptPrinterExtension.java:12]: Pattern is taken form existing SubscriptPrinterExtension and applied "recursively" on group 1 until there is no match. Maybe there is a more elegant way to match all indices. 
There is a testMultiLevelIndexing() method in the PrettyPrintTest class.

```

File: modules/core/res/META-INF/services/edu.kit.iti.algover.term.prettyprint.PrettyPrintExtension (hunk 1 above)
Comment (SpringVaS) [reply modules/core/res/META-INF/services/edu.kit.iti.algover.term.prettyprint.PrettyPrintExtension:10]: Yes.

**Review by mattulbrich — APPROVED** [review]

**Pull request merged**